@smuzaffar   There is a unit error with cmsLHEtoEOSManager.py:
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-b17e60/10242/unitTests.log

Traceback (most recent call last):
  File "/data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_11_2_X_2020-10-23-1100/src/GeneratorInterface/LHEInterface/scripts/cmsLHEtoEOSManager.py", line 238, in <module>
    exeCheckSum = subprocess.Popen(["/afs/cern.ch/cms/caf/bin/cms_adler32",f], stdout=subprocess.PIPE) 
  File "/cvmfs/cms-ib.cern.ch/week1/slc7_amd64_gcc820/external/python/2.7.15-bcolbf2/lib/python2.7/subprocess.py", line 394, in __init__
    errread, errwrite)
  File "/cvmfs/cms-ib.cern.ch/week1/slc7_amd64_gcc820/external/python/2.7.15-bcolbf2/lib/python2.7/subprocess.py", line 1047, in _execute_child
    raise child_exception
OSError: [Errno 2] No such file or directory
+ ERR=1
+ popd
/data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_11_2_X_2020-10-23-1100
+ rm -rf test_cmsLHEtoEOSManager
+ xrdfs root://eoscms.cern.ch/ rm /eos/cms/store/user/cmsbuild/unittest/lhe/ibs/1/1603458798-d3d90891282278bfe1031e7ad4c7688a.lhe.xz
[ERROR] Server responded with an error: [3011] Unable to remove /eos/cms/store/user/cmsbuild/unittest/lhe/ibs/1/1603458798-d3d90891282278bfe1031e7ad4c7688a.lhe.xz; No such file or directory

--- a/GeneratorInterface/LHEInterface/scripts/cmsLHEtoEOSManager.py
+++ b/GeneratorInterface/LHEInterface/scripts/cmsLHEtoEOSManager.py
@@ -235,11 +235,14 @@ if __name__ == '__main__':
         if reallyDoIt and options.compress:
           theList = theCompressedFilesList
         for f in theList:
-            exeCheckSum = subprocess.Popen(["/afs/cern.ch/cms/caf/bin/cms_adler32",f], stdout=subprocess.PIPE) 
-            getCheckSum = subprocess.Popen(["awk", "{print $1}"], stdin=exeCheckSum.stdout, stdout=subprocess.PIPE)
-            exeCheckSum.stdout.close()
-            output,err = getCheckSum.communicate()
-            theCheckSumList.append(output.strip())
+            try:
+                exeCheckSum = subprocess.Popen(["/afs/cern.ch/cms/caf/bin/cms_adler32",f], stdout=subprocess.PIPE)
+                getCheckSum = subprocess.Popen(["awk", "{print $1}"], stdin=exeCheckSum.stdout, stdout=subprocess.PIPE)
+                exeCheckSum.stdout.close()
+                output,err = getCheckSum.communicate()
+                theCheckSumList.append(output.strip())
+            except:
+                theCheckSumList.append("missing-adler32")
 
     newArt = 0
     uploadPath = ''

